### DISCUSSION TITLE
Phase field fracture for hyperelastic materials

### DISCUSSION THREAD

**Discussion by BilelELLAFI** [community/community#28148] [Other Feature Feedback, Questions, & Ideas]
Hello, 
I'm looking please for Phase field UEL subroutine for hyperelastic material or another code that define the phase field fracture and the hyperelastic material both ?

**Comment by jfullstackdev** [comment]
is there a specific project you are mentioning ? that's better if you specify that

**Reply by BilelELLAFI** [reply]
I'm working for contact problem in soft tissu that inlcud a predicting fracture, then i'm looking for UEL hyperelastic phase field fracture with two material one hard and other soft. I used the example of Molnar Gergely "Contact simulation & MATLAB converter" that get the input file modified for UEL input and the phase field UEL fortran, but the result stay in linear elastic or now I want to implement the phase field fracture with hyperelastic material
